splinepy/io/svg.py

```diff
@@ -984,26 +1081,38 @@ def export(
         raise ValueError("fname argument must be string")
 
     # Check if user passed list
-    if (len(splines) == 1) and isinstance(splines[0], list):
-        splines = splines[0]
-
-    for spline in splines:
-        if not isinstance(spline, Spline):
-            raise ValueError("Unexpected spline type")
-        if not 1 <= spline.para_dim <= 2:
-            raise ValueError("Only lines and surfaces supported")
-        if not spline.dim == 2:
-            raise ValueError("Only 2D geometries supported")
+    if (len(objects_to_plot) == 1) and isinstance(objects_to_plot[0], list):
+        objects_to_plot = objects_to_plot[0]
+
+    for object in objects_to_plot:
+        if isinstance(object, Spline):
+            if not 1 <= object.para_dim <= 2:
+                raise ValueError("Only lines and surfaces supported")
+            if not object.dim == 2:
+                raise ValueError("Only 2D geometries supported")
+        elif isinstance(object, _GustafBase):
```

Comment:
```suggestion
        elif isinstance(object, _Vertices):
```
You can just check against vertices. All the other mesh types inherit from vertices